microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,114 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+
+Your administrator can choose to turn on or off the work- or school-related results or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` Microsoft Search in Bing (MSB) policy.  If your administrator turns off the `AddressBarMicrosoftSearchInBingProviderEnabled` policy:
+* In Microsoft Edge on your present device:
+   * You don't see the work- or school- related results from Microsoft Search in Bing.
+   * You see the query suggestions that are returned from your default search engine.
+* In Microsoft Edge on your other devices:
+   * You don't see the work- or school-related results.
+   * You don't see the query suggestions that you previously searched.
+
+You can further protect your privacy by doing any of the following:
+
+* Turn off the suggested queries entirely, by using the toggle **Show me search and site suggestions using my typed characters**, as described in [Controlling whether typed characters are sent](#controlling-whether-typed-characters-are-sent), above.
+
+* Keep the suggested-queries function, but don't send information about the suggested queries to Microsoft.  To prevent sending this information to Microsoft, go to `edge://settings/privacy`, and then under **Search and service improvement**, turn off the toggle **Help improve Microsoft products by sending the results from searches on the web**.  This data is never associated with you or your device, even if you turn on this setting.
+
+* Delete any query suggestion that you don't like, directly in the Address bar, by using the **X** button on the right side when hovering your mouse over the suggestion.
+
+
+<!-- ------------------------------ -->
+#### Sensitive information that's not sent
+
+<!-- The Privacy settings have been updated from what is described here (assume comment means h2 section) -->
+
+If Microsoft Edge detects typing in the Address bar that might contain sensitive information, Microsoft Edge doesn't send the typed text.  Sensitive information includes:
+* Authentication credentials.
+* Local file names.
+* URL data that's normally encrypted.
+
+
+<!-- ------------------------------ -->
+#### Enabling collecting Address bar diagnostic data
+
+You can configure Microsoft Edge to collect diagnostic data about the Address bar.  Collected data includes the number of offered queries for all search providers.  To enable collecting diagnostic data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, and then in **Personalize your web experience**, turn on the setting **Improve your web experience by allowing Microsoft to use your browsing history from this account for personalizing advertising, search, news, and other Microsoft services**.
```

Comment:
Is the number of offered queries the only data that is collected through diagnostic data? If not, it seems a bit dishonest to only call that one out.

"Collected data includes the number of offered queries for all search providers."

Reply:
will discuss offline.  added hidden todo comment.  might Resolve, to handle in future PR. snapshot:
```
You can configure Microsoft Edge to collect diagnostic data 
about the Address bar.  Collected data includes the number 
of offered queries for all search providers.<!-- todo: is the 
number of offered queries the only data that's collected through 
diagnostic data?  if not, shouldn't list that one only -->  To 
enable collecting diagnostic data, select **Settings and more** 
(![Settings and more icon](./index-images/settings-and-more-
icon.png)) > **Settings** > **Privacy, search, and services**, 
and then in the **Personalization & advertising** section, turn 
on the setting **Allow Microsoft to save your browsing activity 
including history, usage, favorites, web content, and other 
browsing data to personalize Microsoft Edge and Microsoft 
services like ads, search, shopping and news.**
```

Reply:
added hidden todo cmt, Resolving, to answer in future PR